### DISCUSSION TITLE
Problem only with Thumbnail view in Explorer

### DISCUSSION THREAD

**Discussion by donatopace** [d2phap/ImageGlass#1562] [General]
I have two images "jxl" 
and both are seen under IMAGEGLASS
but only one of two has thumbnail problem.

I have also installed "SageThumbs"
but o have not solved problem.
I attached images

![jxl Thumbnail missing](https://user-images.githubusercontent.com/43314459/234360550-d060d0ec-9698-4c2c-9074-023433dc195b.png)

**Comment by d2phap** [comment]
ImageGlass does not provide thumbnail extension for Explorer. This is probably issue with SageThumbs.

**Reply by donatopace** [reply]
Any Best alternative to SageThumbs?
Thanks.

**Comment by donatopace** [comment]
Found Solution Works well:
https://github.com/saschanaz/jxl-winthumb
